hudi-utilities/src/main/java/org/apache/hudi/utilities/config/SqlTransformerConfig.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.utilities.config;
+
+import org.apache.hudi.common.config.ConfigClassProperty;
+import org.apache.hudi.common.config.ConfigGroups;
+import org.apache.hudi.common.config.ConfigProperty;
+import org.apache.hudi.common.config.HoodieConfig;
+
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * DeltaStreamer SQL Transformer Configs
+ */
+@Immutable
+@ConfigClassProperty(name = "DeltaStreamer SQL Transformer Configs",
+    groupName = ConfigGroups.Names.DELTA_STREAMER,
```

Comment:
is this DELTA_STREAMER config or DELTA_STREAMER_SQL_TRANSFORMER_CONFIG?

Reply:
how are we envisioning the configs for diff sources in Deltastreamer? I don't think we can have one config class per source.

Reply:
oh, my bad. group name is DELTA_STREAMER. makes sense.